DeviceDiagnostics/doc/DeviceDiagnosticsPlugin.md

```diff
@@ -60,7 +60,7 @@ The table below provides and overview of terms and abbreviations used in this do
 <a name="head.Description"></a>
 # Description
 
-The `DeviceDiagnostics` plugin provides an interface for the WebPA service for this device only. It allows you to get device configurations.
+The `DeviceDiagnostics` plugin provides an interface for the WebPA service for this device only. It allows you to get device configurations and AV decoder status.
```

Comment:
Pls change this to "The DeviceDiagnostics Plugin provides additional diagnostics information. This includes device configuration and AV decoder status"